pallets/xcm-core-buyer/src/mock.rs

```diff
@@ -161,6 +164,18 @@ parameter_types! {
     pub const ParachainId: ParaId = ParaId::new(1000);
 }
 
+parameter_types! {
+    pub const CoreBuyingXCMQueryTtl: u32 = 100;
```

Comment:
what is the difference between these 2?

Reply:
So, the first `CoreBuyingXCMQueryTtl ` is used in the xcm query itself. If the query response comes  from the relay chain after this `Ttl`, it will be ignored and notification will not be triggered.

The other `ttl` is the additional `ttl` on top of the `CoreBuyingXCMQueryTtl ` to indicate when to clear the in flight order and consider the result as unknown.